hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -3,11 +3,189 @@
+          <div className="card text-center">
+            <div className="card-header text-white bg-success">
+              <h4 className="mb-0">Dog Media</h4>
+            </div>
+            <div className="card-body text-left">
+              <p className="card-text">
+                <h5>Labels</h5>
+                <ul>
+                  <li>dog</li>
+                  <li>review</li>
+                </ul>
+                <h5>Matching</h5>
+                <ul>
+                  <li>PDQ: Distance 14</li>
+                  <li>Video MD5</li>
+                </ul>
+              </p>
+            </div>
+            <a href="#todo" className="btn btn-primary">
+              Edit
+            </a>
+          </div>
+        </div>
+      </Row>
+    </>
+  );
+}
+
+function DataSet() {
+  return (
+    <>
+      <Col lg={4} sm={6} xs={12} className="mb-4">
+        <Card className="text-center">
+          <Card.Header className="text-white bg-success">
+            <h4 className="mb-0">Collboration on Cat Imagery</h4>
+          </Card.Header>
+          <Card.Subtitle className="mt-0 mb-0 text-muted">Shared</Card.Subtitle>
+          <Card.Body className="text-left">
+            <Card.Text>
+              <h5>Labels</h5>
+              <ul>
+                <li>cat</li>
+                <li>unverified</li>
+                <li>*low_precision</li>
+              </ul>
+              <h5>Matching</h5>
+              <ul>
+                <li>PDQ: Distance 17</li>
+              </ul>
+            </Card.Text>
+          </Card.Body>
+          <ListGroup variant="flush" className="text-left">
+            <ListGroup.Item>
+              <h5>Shared From: Cat Dataset</h5>
+              <h6>Label Map</h6>
+              <ul>
+                <li>low_precision → *</li>
+              </ul>
+            </ListGroup.Item>
+          </ListGroup>
+          <a href="#todo" className="btn btn-primary">
+            Edit
+          </a>
+        </Card>
+      </Col>
+    </>
+  );
+}
+
+function ExternalSignalSources() {
+  return (
+    <>
+      <h2>External Signal Sources</h2>
+      <SignalSource />
+    </>
+  );
+}
+
+function SignalSource() {
+  return (
+    <>
+      <Row className="mt-3">
+        <Col lg={4} sm={6} xs={12} className="mb-4">
+          <Card className="text-center">
+            <Card.Header className="text-white bg-success">
+              <h4 className="mb-0">Cat Dataset</h4>
+            </Card.Header>
+            <Card.Subtitle className="mt-0 mb-0 text-muted">
+              ThreatExchange
+            </Card.Subtitle>
+            <Card.Body className="text-left">
+              <Card.Text>
+                <div>
+                  <h5 className="mb-0">Privacy Group</h5>
+                  <div>Cat Image Collaboration</div>
+                </div>
+                <div className="mt-4">
+                  <h5 className="mb-0">Settings</h5>
+                  <div>✔️ Seen status</div>
```

Comment:
In addition to iconography, another additional way to indicate on / off or selected / not selected is bold vs not. These days bold is done with `<strong>this will be bold</strong>`.

Reply:
Tried it out, and thought that it might generate confusion, think icon or bold is enough, and think icon is clearer. 
![image](https://user-images.githubusercontent.com/1654004/112520938-2fe3dc00-8d59-11eb-9175-86679d55e77d.png)